concordium-node/src/plugins/consensus.rs

```diff
@@ -271,7 +271,7 @@ pub fn handle_consensus_inbound_msg(
         }
 
         // relay external messages to Consensus
-        let consensus_result = send_msg_to_consensus(node, source, consensus, &request)?;
+        let (consensus_result, _) = send_msg_to_consensus(node, source, consensus, &request)?;
```

Comment:
It seems wrong that we throw away the finalizer like this.

Reply:
Yes. I fixed this. Note this branch of `no_rebroadcast_consensus_validation` will be removed in a subsequent PR.